<?xml version="1.0" encoding="UTF-8"?>
    <source package="javax.management.relation">
        <class name="InvalidRelationIdException" line="28">
            <extends class="RelationException"/>
            <comment line="37">
                                
            </comment>
            <javadoc line="28">
                This exception is raised when relation id provided for a relation is already
                  used.                
                <since>
                    1.5                    
                </since>
            </javadoc>
            <declaration name="serialVersionUID" type="long" line="37"/>
            <javadoc line="39">
                Default constructor, no message put in exception.                
            </javadoc>
            <method name="InvalidRelationIdException" type="constructor" line="42"/>
            <javadoc line="46">
                Constructor with given message put in exception.                
                <param>
                    message the detail message.                    
                </param>
            </javadoc>
            <method name="InvalidRelationIdException" type="constructor" line="51">
                <params>
                    <param name="message" type="String"/>
                </params>
            </method>
        </class>
    </source>